app/Models/User.php

```diff
@@ -540,7 +540,7 @@ public function rank()
 
     public function rankHistories()
     {
-        return $this->hasMany(RankHistory::class);
+        return $this->hasMany("App\Models\RankHistory", 'user_id', 'user_id');
```

Comment:
It is using new style. Also the default already understands the column name.

What exactly the error you're entering?